### PR TITLE
plutil: add page

### PR DESCRIPTION
**Pull request by mflint**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/osx/plutil.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# plutil
+
+> View, convert, validate, or edit property list ("plist") files.
```
Comment (navarroaxel) [inline pages/osx/plutil.md:3]: Note: I found some man pages like https://www.theiphonewiki.com/wiki/Plutil#Apple.27s_plutil but is not synced with the mac options.

File: pages/osx/plutil.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Dump the contents of plist files in human-readable format:
+
+`plutil -p {{file1.plist}} {{file2.plist}}`
```
Comment (navarroaxel) [inline pages/osx/plutil.md:7]: We use this format:
```suggestion
`plutil -p {{path/to/file1.plist}} {{path/to/file2.plist}}`
```

File: pages/osx/plutil.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`plutil -convert xml1 {{file1.plist}} {{file2.plist}}`
+
+- Convert plist files to binary format, overwriting the original files in-place:
+
+`plutil -convert binary1 {{file1.plist}} {{file2.plist}}`
+
+- Convert a single plist file to a different format, writing to a new file:
+
+`plutil -convert {{xml1|binary1|json|swift|objc}} {{file.plist}} -o {{new_file.plist}}`
```
Comment (navarroaxel) [inline pages/osx/plutil.md:19]:
```suggestion
`plutil -convert {{xml1|binary1|json|swift|objc}} {{path/to/file.plist}} -o {{path/to/new_file.plist}}`
```

File: pages/osx/plutil.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- Convert a single plist file to a different format, writing to stdout:
+
+`plutil -convert {{xml1|binary1|json|swift|objc}} {{file.plist}} -o -`
```
Comment (navarroaxel) [inline pages/osx/plutil.md:23]:
```suggestion
`plutil -convert {{xml1|binary1|json|swift|objc}} {{path/to/file.plist}} -o -`
```

File: pages/osx/plutil.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+- Convert plist files to binary format, overwriting the original files in-place:
+
+`plutil -convert binary1 {{file1.plist}} {{file2.plist}}`
+
+- Convert a single plist file to a different format, writing to a new file:
+
+`plutil -convert {{xml1|binary1|json|swift|objc}} {{file.plist}} -o {{new_file.plist}}`
+
+- Convert a single plist file to a different format, writing to stdout:
```
Comment (navarroaxel) [inline pages/osx/plutil.md:21]:
```suggestion
- Convert a plist file to a different format, writing to `stdout`:
```

File: pages/osx/plutil.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+- Dump the contents of plist files in human-readable format:
+
+`plutil -p {{file1.plist}} {{file2.plist}}`
+
+- Convert plist files to XML format, overwriting the original files in-place:
+
+`plutil -convert xml1 {{file1.plist}} {{file2.plist}}`
+
+- Convert plist files to binary format, overwriting the original files in-place:
```
Comment (navarroaxel) [inline pages/osx/plutil.md:13]:
```suggestion
- Convert one or more space-separated plist files to binary format, overwriting the original files in-place:
```

File: pages/osx/plutil.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+# plutil
+
+> View, convert, validate, or edit property list ("plist") files.
+
+- Dump the contents of plist files in human-readable format:
+
+`plutil -p {{file1.plist}} {{file2.plist}}`
+
+- Convert plist files to XML format, overwriting the original files in-place:
```
Comment (navarroaxel) [inline pages/osx/plutil.md:9]:
```suggestion
- Convert one or more space-separated plist files to XML format, overwriting the original files in-place:
```

File: pages/osx/plutil.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+`plutil -convert xml1 {{file1.plist}} {{file2.plist}}`
+
+- Convert plist files to binary format, overwriting the original files in-place:
+
+`plutil -convert binary1 {{file1.plist}} {{file2.plist}}`
+
+- Convert a single plist file to a different format, writing to a new file:
```
Comment (navarroaxel) [inline pages/osx/plutil.md:17]:
```suggestion
- Convert a plist file to a different format, writing to a new file:
```

File: pages/osx/plutil.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
+# plutil
+
+> View, convert, validate, or edit property list ("plist") files.
+
+- Dump the contents of plist files in human-readable format:
```
Comment (navarroaxel) [inline pages/osx/plutil.md:5]: Maybe this description is more readable
```suggestion
- Display the contents of one or more space-separated plist files in human-readable format:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @mflint, Thanks for your submission!

I made some comments

**Comment by mflint** [comment]
Thank-you for your suggestions @navarroaxel. I'm not sure that "space separated" is needed, as this would violate the guidance to "avoid explaining general UNIX concepts that could apply to any command". But the rest is great - thank-you!

**Comment by navarroaxel** [comment]
> Thank-you for your suggestions @navarroaxel. I'm not sure that "space separated" is needed, as this would violate the guidance to "avoid explaining general UNIX concepts that could apply to any command". But the rest is great - thank-you!

I'm not sure, several pages say that. maybe @sbrl or @mebeim could check this

**Review by navarroaxel — APPROVED** [review]

**Comment by mebeim** [comment]
Hi there @navarroaxel thanks for the ping. I think there is no problem if we avoid saying "space separated" when it's already obvious from the example, I.E. the example contains `{{file1 file2 ...}}`. We prefer using `{{file1 file2 ...}}` instead of `{{file1}} {{file2}}` in such cases (or basically almost any case where the descriptions applies to "one or more files"). Same goes for `{{path/to/file1.plist}} {{path/to/file1.plist}}`, you can just make that `{{file1.plist file2.plist ...}}`. The `path/to/file` form is preferred when there is only one file or two different files with different meanings (e.g. one input and one output).

File: pages/osx/plutil.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
+# plutil
+
+> View, convert, validate, or edit property list ("plist") files.
+
+- Display the contents of one or more plist files in human-readable format:
+
+`plutil -p {{path/to/file1.plist}} {{path/to/file2.plist}}`
```
Comment (mebeim) [inline pages/osx/plutil.md:7]:
```suggestion
`plutil -p {{file1.plist file2.plist ...}}`
```

File: pages/osx/plutil.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+
+- Convert one or more plist files to XML format, overwriting the original files in-place:
+
+`plutil -convert xml1 {{path/to/file1.plist}} {{path/to/file2.plist}}`
```
Comment (mebeim) [inline pages/osx/plutil.md:11]:
```suggestion
`plutil -convert xml1 {{file1.plist file2.plist ...}}`
```

File: pages/osx/plutil.md [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+
+- Convert one or more plist files to binary format, overwriting the original files in-place:
+
+`plutil -convert binary1 {{path/to/file1.plist}} {{path/to/file2.plist}}`
```
Comment (mebeim) [inline pages/osx/plutil.md:15]:
```suggestion
`plutil -convert binary1 {{file1.plist file2.plist ...}}`
```

**Review by mebeim — CHANGES_REQUESTED** [review]
Suggestions from my above comment.

**Comment by mflint** [comment]
Thank-you for the feedback @mebeim :)

**Review by mebeim — APPROVED** [review]

**Pull request merged**